### PR TITLE
1.42.4 fix pg vulnerability

### PR DESCRIPTION
**Pull request by oleurud**
Adding a new cartodb-psql fixing a pg vulnerability

This will be the tag 1.42.4-3

-----------------

More info about pg vulnerability 
https://node-postgres.com/announcements#2017-08-12-code-execution-vulnerability

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "dependencies": {
     "bintrees": "1.0.1",
     "bunyan": "1.8.1",
-    "cartodb-psql": "~0.6.0",
+    "cartodb-psql": "0.6.2",
```
Comment (rochoa) [inline package.json]: Do we really need to change this, or is it enough to regenerate the shrinkwrap file?

Comment (oleurud) [reply package.json]: You are right, I have changed the package.js updating the cartodb-psql to "~0.6.0"

File: NEWS.md [hunk 2]
```diff
@@ -3,6 +3,11 @@
 ## 1.42.5
 Released 2016-mm-dd
 
+## 1.42.4-3
+Released 2017-08-31
```
Comment (dgaubert) [inline NEWS.md]: I'd try to wait to release time to specify the date of the release.

Comment (oleurud) [reply NEWS.md]: Sorry, I have not followed the process well in all the pg vulnerability PRs. I have always forgotten something.

I will pay more attention in the PR process

**Comment by dgaubert** [comment]
🚀 :shipit:

**Comment by oleurud** [comment]
The tag 1.42.4-3 was created

**Pull request merged**